tests/integration_tests/test_embeddings.py

```diff
@@ -16,14 +17,15 @@
 class ModelSpec:
     deployment: EmbeddingsDeployment
     default_dimensions: int
-    supports_dimensions: bool
+    supports_dimensions: bool = False
```

Comment:
For discussion:

This example made me think of this naming:

Cohere **support dimensions**, but it doesn't support dimensions parameter in request

Maybe let's name it something like `support_dimensions_parameter`, and remove this default value, to make config more explicit?

Becuse now it's kinda confusing, because when I saw
```
    ModelSpec(
        ...
        default_dimensions=1024,
        ...
    ),
``
the same moment I thought that it support dimensions

Reply:
Fixed, I added comments to the config fields and removed defaults.